File: src/service/PersonnelService.js
import PersonnelApi from '../network/api/PersonnelApi'

export default {
    getPersonnelList (projectId, success, error, failure) {
        PersonnelApi.getPersonnelList(projectId, success, error, failure)
    },
    addPersonnel (personnel, success, error, failure) {
        PersonnelApi.addPersonnel(personnel, success, error, failure)
    },
    updatePersonnel (personnel, success, error, failure) {
        PersonnelApi.updatePersonnel(personnel, success, error, failure)
    },
    removePersonnel(personnelId, success, error, failure) {
        PersonnelApi.removePersonnel(personnelId, success, error, failure)
    }
}

File: defectmanager-server/src/main/java/group/geolo/defectmanager/service/PersonnelService.java
package group.geolo.defectmanager.service;

import group.geolo.defectmanager.entity.Personnel;

import java.util.List;

/**
 * @author 桀骜(Geolo)
 * @version 1.0
 * @date 2019/10/12
 */
public interface PersonnelService {

    Personnel getPersonnel(Integer personnelId);

    List<Personnel> getPersonnels(Integer projectId);

    void addPersonnel(Personnel personnel);

    void updatePersonnel(Personnel personnel);

    void removePersonnel(Integer personnelId);

    List<Personnel> getUserPersonnelOfProject(Integer userId, Integer projectId);
}
